examples/lib/stories/controls/scroll.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/palette.dart';
+import 'package:flame/extensions.dart';
+
+// TODO(luan) doesnt seem to be working, figure it out
```

Comment:
Still valid comment?

Reply:
yes, it doesnt work (I left a huge comment with all the problems in the examples for people to take)